lib/core/dispatchRequest.js

```diff
@@ -48,7 +49,7 @@ module.exports = function dispatchRequest(config) {
   );
 
   utils.forEach(
-    ['delete', 'get', 'head', 'post', 'put', 'patch', 'common'],
+    ['delete', 'get','head',  'post', 'put', 'patch', 'common'],
```

Comment:
format: style mismatch:
```suggestion
['delete', 'get','head',  'post', 'put', 'patch', 'common'], function cleanHeaderConfig(method) {
```

`16b835cf` ⏎␣⁺␣⁺ at column 17 should be removed.
`a618c6b3` Replace ⏎ with ␣ at column 65.